lib/stdlib/doc/src/peer.xml

```diff
@@ -0,0 +1,319 @@
+      documentation). A peer node started on the same host
+      as the origin inherits certain environment values from the origin,
+      such as the current directory and the environment variables. For
+      what can be assumed about the environment when a peer is started
+      on another host, see the documentation for the <c>ssh</c>
+      program.</p>
+
+    <p>Peer node terminates when it loses connection to the origin.
+      This connection could be Erlang distribution connection (default),
+      or alternative like TCP or standard IO (stdin/stdout). Alternative
+      connection provides a way to test the distribution itself.
+    </p>
+
+    <p>This module is designed to facilitate multi-node testing.
+    </p>
+
+  </description>
+
+  <datatypes>
+    <datatype>
+      <name name="start_options"/>
+      <desc>
+        <p>
+          Options that can be used when starting
+          a <c>peer</c> node through <seemfa marker="#start/1"><c>start/1</c></seemfa>
+          and <seemfa marker="#start_link/0"><c>start_link/0,1</c></seemfa>.
+        </p>
+        <taglist>
+          <tag><c>name</c></tag>
+          <item>
+            <p>Node name (part before "@").</p>
+          </item>
+          <tag><c>host</c></tag>
+          <item>
+            <p>Host to start the peer node at. When not specified, peer is
+              started on the same host as the origin.</p>
+          </item>
+          <tag><c>longnames</c></tag>
+          <item>
+            <p>Use long names to start a node. Default is taken from origin
+              using <c>net_kernel:longnames()</c>.</p>
+          </item>
+          <tag><c>remote</c></tag>
+          <item>
+            <p>Use remote shell to start the node. When requested <c>host</c>
+              is different from the origin host, and <c>remote</c> is not specified,
+              it is implied to be <c>rsh</c>.</p>
+          </item>
+          <tag><c>report_reason</c></tag>
+          <item>
+            <p>When set to false (default), peer node termination changes the
+              status reported by controlling process. When set to true,
+              peer shutdown causes controlling process to exit with the
+              <c>Reason</c> of the peer node termination.</p>
+          </item>
+          <tag><c>progname</c></tag>
+          <item>
+            <p>Path to "erl", by default taken from <c>init:get_argument(progname)</c>.
+              If progname is not known, <c>peer</c> makes best guess given current
+              ERTS version.
+            </p>
+            <p>
+              Useful for testing backwards compatibility with previous releases,
+              or when Erlang installation location is missing from the <c>PATH</c>,
+              or when starting peers on a host with different operating system.
+            </p>
+          </item>
+          <tag><c>connection</c></tag>
+          <item>
+            <p>Alternative connection specification.</p>
```

Comment:
What is the default value?

Reply:
Default (when no option is specified) is Erlang Distribution connection only, just like existing `slave` implementation.